src/components/PresenceMap/component.js

```diff
@@ -0,0 +1,141 @@
+import React, { useRef, useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import getMapContainerStyles from './getMapContainerStyles';
+import StyledContainer from './styled/container';
+import StyledMapContainer from './styled/mapContainer';
+
+export default function MapComponent(props) {
+    const container = useRef();
+    const mapContainer = useRef();
+    const [markers, setMarker] = useState([]);
```

Comment:
`setMarkers` ?? although I dont' think we need state for the markers since they can be compute when needed based on the objects